Content.Client/Instruments/UI/InstrumentBoundUserInterface.cs

```diff
@@ -103,7 +104,13 @@ public void OpenChannelsMenu()
             _channelsMenu ??= new ChannelsMenu(this);
             EntMan.TryGetComponent(Owner, out InstrumentComponent? instrument);
 
-            _channelsMenu.Populate(instrument);
+            ActiveInstrumentComponent? activeInstrument = null;
+            if (instrument is { Master: not null })
+                EntMan.TryGetComponent(instrument.Master, out activeInstrument);
```

Comment:
Masters can actually be chained infinitely AFAIK. Not sure if you need to account for that?